types/validator_set_test.go

```diff
@@ -161,7 +161,7 @@ func TestValidatorSetValidateBasic(t *testing.T) {
 				Proposer:           badValNoPublicKey,
 			},
 			err: true,
-			msg: "invalid validator pub key #0: validator does not have a public key",
+			msg: "thresholdPublicKey error: validator public key is not set",
```

Comment:
Why do we expect an error that thresholdPublicKey is missing when we do not pass validator private key?